src/main/java/creativei/vo/ResponseObject.java

```diff
@@ -27,28 +27,17 @@ public void setStatus(ResponseStatus status) {
         this.status = status;
     }
 
-<<<<<<< .mine
-    public Object getData(){return data;}
-
-
-
-=======
-<<<<<<< HEAD
     public Object getData() {
         return data;
     }
->>>>>>> .theirs
-
-    public void setData(Object data) {
-        this.data = data;
-=======
-    public Object getData(){return data;}
 
-    public void setData(Object data) {
-        this.data = data;
->>>>>>> inquiry_register
+    public Error getException() {
+        return exception;
     }
 
+    public ResponseObject(){
```

Comment:
No need of default constructor for this class. We will always use parameterized constructors.